README.md

```diff
@@ -69,90 +78,104 @@ An example of a successful response is:
+If your organization uses multiple Active Directory tenants, log into the same tenant used to [Register a New Office 365 Web Application](#register-a-new-office-365-web-application). To find your Office 365 tenant ID, see [Find your Office 365 tenant ID](https://support.office.com/en-gb/article/find-your-office-365-tenant-id-6891b561-a52d-4ade-9f39-b492285e2c9b).
 
-### Deploy via the Custom ARM Template in an Azure Subscription
+### Deploy with the Custom ARM Template in an Azure Subscription
+
+Click the button below to start deployment. 
 
 [![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Falertlogic%2Fazure-collector%2Fmaster%2Ftemplate.json)
 
-Fill in required template parameters and click the `Purchase` button to start a deployment:
-   - `Name` - This is the name of the log source that will show in the Alert Logic UI
-   - `Storage Name` - Any Storage Account name (that does not currently exist)
-   - `Alert Logic Access Key ID` - `access_key_id` returned from AIMs [above](#create_an_alert_logic_access_key)
-   - `Alert Logic Secret Key` - `secret_key` returned from AIMs [above](#create_an_alert_logic_access_key)
-   - `Alert Logic API endpoint` - leave it as `api.global-services.global.alertlogic.com`
-   - `Alert Logic Data Residency` - leave it as `default`
-   - `Office365 Content Streams` - The list of streams you would like to collect.  Valid values are:
-        - ["Audit.AzureActiveDirectory","Audit.Exchange","Audit.SharePoint","Audit.General", "DLP.All"]
-   - `Service Principal ID` - The `Object ID` of the application that created the subscription. You can obtain it from _Azure_ -> _AD_ -> _App registrations_ -> _Your app name_ -> Link under _Managed application in local directory_ -> _Properties_ -> _Object ID_
-   - `App Client ID` - The GUID of your application that created the subscription. You can obtain it from _Azure_ -> _AD_ -> _App registrations_ -> _Your app name_
+1. Provide the following required template parameters and click the `Purchase` button to start a deployment:
+   - `Name` - Type the name of the log source to appear in the Alert Logic console.
+   - `Storage Name` - Any storage account name (that does not currently exist).
+   - `Alert Logic Access Key ID` - The `access_key_id` you created above
+   - `Alert Logic Secret Key` - The `secret_key` you created above.
+   - `Alert Logic API endpoint` - Leave the default value (api.global-services.global.alertlogic.com).
+   - `Alert Logic Data Residency` - Leave the value as `default`.
+   - `Office365 Content Streams` - The log types you want to collect. Valid values are:
+      ["Audit.AzureActiveDirectory","Audit.Exchange","Audit.SharePoint","Audit.General", "DLP.All"]
```

Comment:
Could you please remove `DLP.All` from this list.
We haven't managed to remove it previously.